src/pybind/mgr/prometheus/module.py

```diff
@@ -598,8 +597,8 @@ def get_rbd_stats(self):
         pools_refreshed = False
         if pools:
             next_refresh = self.rbd_stats['pools_refresh_time'] + \
-                self.get_localized_module_option(
-                    'rbd_stats_pools_refresh_interval', 300)
+                           self.get_localized_module_option(
```

Comment:
what is this change for?

Reply:
indentation; to line up the right side of the =

Reply:
I'll remove the change

Reply:
I think it's correct, though, no?

Reply:
@dmick yes, it's correct. but i don't think we should encourage this kind of reformatting changes. as it is not specified by a guideline or a PEP AFAICT.

@iliul first, i don't think it is required by PEP8.

and i guess the current indention is what i get with my Emacs with default (well, mostly) settings. and probably is what other developers will have with their editors.

if we **really** care about the indentation here. i'd suggest add quote the expression at the right side of `=` with parentheses, like
```py
            next_refresh = (self.rbd_stats['pools_refresh_time'] +
                            self.get_localized_module_option(
                                'rbd_stats_pools_refresh_interval', 300))
```

i don't feel strong either way. but i still suggest drop this change, personally.

Reply:
I'll defer to deeper research; I didn't read PEP8, I just assumed the change was suggested in compliance with PEP8.  Whatever.

Reply:
we can run `pep8` or `pycodestyle` against this source file. presumably, it should find out all PEP8 issues.

Reply:
@tchaikov OK, Use [`autopep8`](https://github.com/hhatto/autopep8) checked again, command is:
```shell
# autopep8 --in-place prometheus/module.py mgr_module.py mgr_util.py hello/module.py
```

Only the files related to this PR, others should be put into the next job( NEW PR).